lib/mbta.js

```diff
@@ -1,11 +1,35 @@
 'use strict'
 
 // Code here.
+const mbtaSubway = {
+  Red: ['South Station', 'Park Street', 'Kendall', 'Central', 'Harvard', 'Porter', 'Davis', 'Alewife'],
+  Green: ['Government Center', 'Park Street', 'Boylston', 'Arlington', 'Copley', 'Hynes', 'Kenmore'],
+  Orange: ['North Station', 'Haymarket', 'Park Street', 'State', 'Downtown Crossing', 'Chinatown', 'Backbay', 'Forest Hills']
+}
+let startIndex
```

Comment:
You want to initialize these variables inside your function, rather than outside